apps/dashboard/lib/dashboard_web/live/start_live.ex

```diff
@@ -20,6 +20,17 @@ defmodule DashboardWeb.Live.StartLive do
             <% end %>
           </span>
         </div>
+        <div>
+          <%= if @state.status == :finish and not is_nil(@state.validation_status) do %>
+            <div><strong> Validation Status: </strong>
+              <%= if @state.validation_status == :passed do %>
+                <span class="validation-status"> Passed!</span>
+              <% else %>
+                <span class="validation-status error-status"> Failed!(Check logs) </span>
+              <% end %>
+            </div>
+          <% end %>
+        </div>
```

Comment:
```suggestion
        <div>
          <%= if not is_nil(@state.validation_status) do %>
            <div>
                <strong> Validation Status: </strong>
                <span class="validation-status ">@state.validation_status</span>
            </div>
          <% end %>
        </div>
```

   Add error-status class dynamically if validation-status is failed!.